1035-tooltip-design-guidance.md

```diff
@@ -0,0 +1,29 @@
+---
+category: component design guidance
+state: draft
+created: 2024-10-18
+updated: 2024-10-22
+---
+
+# Tooltip Component
+
+[Tooltip](https://clarity.design/documentation/tooltip) component is a brief, informative message or short description about an element.
+
+## Guidance
+
+- **Must not** use tooltips on components. Instead, provide a descriptive label and use inline or signpost help for more information.
```

Comment:
I'm not sure what this means.

Reply:
In my sense, you should add tooltip over any of the components like select/input component. Instead, have a label next to it and tag tooltip.

Reply:
This message is available at clarity.design as well.

Reply:
Jay has added more details. Hence resolving.